src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started working.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stop working.");
```

Comment:
There's a typo in the message: 'Truck stop working.' should be 'Truck stopped working.' to maintain consistency with the past tense used in the `doWork` method.